### PR TITLE
change appearance validate and cancel button

### PR DESCRIPTION
**Pull request by anistouri**

### CODE REVIEW

**Review by YenguiSeddik — APPROVED** [review]

File: src/components/utils/popup-confirmation-dialog.js [hunk 1]
```diff
@@ -26,9 +27,9 @@ const PopupConfirmationDialog = ({
                 </DialogContentText>
             </DialogContent>
             <DialogActions>
-                <Button onClick={() => setOpenConfirmationPopup(false)}>
+                <CancelButton onClick={() => setOpenConfirmationPopup(false)}>
                     <FormattedMessage id="cancel" />
```
Comment (flomillot) [inline src/components/utils/popup-confirmation-dialog.js]: There is already `<FormattedMessage id="cancel" /> `in `CancelButton` no ?

File: src/components/dialogs/delete-dialog.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ const styles = {
     tooltip: {
         maxWidth: '1000px',
     },
+    deleteButtonTextColor: (theme) => ({
+        color: theme.palette.text.primary,
```
Comment (flomillot) [inline src/components/dialogs/delete-dialog.js]: Don't use `sx` and use `color` instead

File: src/components/dialogs/replace-with-script-dialog.js [hunk 3]
```diff
@@ -48,10 +49,12 @@ const ReplaceWithScriptDialog = ({ id, open, onClose, onClick, title }) => {
...
-                <Button onClick={handleClose} variant="outlined">
-                    <FormattedMessage id="cancel" />
-                </Button>
-                <Button onClick={handleClick}>
+                <CancelButton
+                    onClick={handleClose}
+                    variant="outlined"
+                    withCustomColor={true}
+                />
+                <Button onClick={handleClick} color={'customButton'}>
```
Comment (flomillot) [inline src/components/dialogs/replace-with-script-dialog.js:57]: Validate in grey too ?

File: src/components/dialogs/delete-dialog.js [hunk 4]
```diff
@@ -210,14 +210,17 @@ const DeleteDialog = ({
...
+                <Button
+                    onClick={handleClick}
+                    disabled={loadingState}
+                    color={'customButton'}
                 >
-                    <FormattedMessage id="cancel" />
-                </Button>
-                <Button onClick={handleClick} disabled={loadingState}>
                     {(loadingState && <CircularProgress size={24} />) || (
                         <FormattedMessage id="delete" />
```
Comment (flomillot) [inline src/components/dialogs/delete-dialog.js:225]: In the US they don't specify to change the delete button. If so, we should do it everywhere no ?

File: src/components/dialogs/replace-with-script-dialog.js (hunk 3 above)
Comment (anistouri) [reply src/components/dialogs/replace-with-script-dialog.js:57]: When the cancel button is customized, the secondary button should be white on the dark theme and black on the light theme

File: src/components/dialogs/delete-dialog.js (hunk 4 above)
Comment (anistouri) [reply src/components/dialogs/delete-dialog.js:225]: After discussion with the PO, we are implementing it here and this component is generic and used wherever there is a deletion

**Review by flomillot — APPROVED** [review]

**Pull request merged**